lib/recurly/token.js

```diff
@@ -160,10 +160,12 @@ function token (customerData, bus, done) {
       }));
     }
 
-    const bin = inputs.number.substr(0, 6);
-    Risk.preflight({ recurly: this, bin })
+    const { number, month, year } = inputs;
+    Risk.preflight({ recurly: this, number, month, year })
       .then(results => inputs.risk = results)
-      .then(() => this.request.post({ route: '/token', data: inputs, done: complete }))
+      .then(() =>  {
+        this.request.post({ route: '/token', data: inputs, done: complete })
```

Comment:
this should be returned to retain the existing behavior.

nit: needs a semicolon at EOL